packages/feedback/src/widget/Actor.css.ts

```diff
@@ -0,0 +1,94 @@
+import type { FeedbackTheme } from '../types';
+
+/**
+ * Creates <style> element for widget actor (button that opens the dialog)
+ */
+export function createActorStyles(d: Document, theme: FeedbackTheme): HTMLStyleElement {
+  const style = d.createElement('style');
+  style.textContent = `
```

Comment:
i'll mention this for the coolness factor:
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/CSSStyleSheet

But i haven't looked at browser compat or perf or anything. Just here for completness.

Reply:
I think this is a bit easier to read, not sure if CSSStyleSheet offers any other benefits for mostly static CSS